uo_sys_framework/StartInParachuteModule/cfgFunctions/common/fn_DoParachute.sqf

```diff
@@ -9,30 +10,28 @@ _stear = false;
 _class = "NonSteerable_Parachute_F";
 _doPara = false;
 
-if (count _this > 2) then 
+if (count _this > 2) then
```

Comment:
All these checks can be substituted with the command `params` and default values